packages/rum_sdk/example/lib/main.dart

```diff
@@ -11,32 +11,30 @@ void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   HttpOverrides.global = RumHttpOverrides(HttpOverrides.current);
   await dotenv.load(fileName: ".env");
-  RumFlutter().transports.add(
-      OfflineTransport(maxCacheDuration: const Duration(days: 3))
-  );
-  await RumFlutter().runApp( optionsConfiguration:
-    RumConfig(
-      appName: "example_app",
-      appVersion: "2.0.1",
-      appEnv: "Test",
-      apiKey: "api_key",
-      anrTracking: true,
-      cpuUsageVitals: true,
-      collectorUrl: "http://10.0.2.2:8027/collect",
-      enableCrashReporting: true,
-      memoryUsageVitals: true,
-      refreshRateVitals: true,
-      fetchVitalsInterval: const Duration(seconds: 30),
-    ),
-    appRunner: () async {
-      runApp( DefaultAssetBundle(bundle: RumAssetBundle(), child: const RumUserInteractionWidget(child: MyApp()))
-      );
-    }
-  );
+  RumFlutter()
+      .transports
+      .add(OfflineTransport(maxCacheDuration: const Duration(days: 3)));
+  await RumFlutter().runApp(
+      optionsConfiguration: RumConfig(
+        appName: "example_app",
+        appVersion: "2.0.1",
+        appEnv: "Test",
+        apiKey: dotenv.env['FARO_API_KEY'] ?? '',
+        anrTracking: true,
+        cpuUsageVitals: true,
+        collectorUrl: dotenv.env['FARO_COLLECTOR_URL'] ?? '',
```

Comment:
I guess here you want to use the data located in your .env file?
same for the `apiKey`